src/sgx/libpoet_bridge/enclave.cpp

```diff
@@ -542,17 +539,23 @@ namespace sawtooth {
             sgx_ec256_signature_t* outWaitCertificateSignature
```

Comment:
Can we declare a structure for initialize and finalize wait certificate. Too many function parameters makes the function signature almost non maintainable.